infra/production/base/module/ecs/capacity_provider.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_launch_template" "service_spot_launch_template" {
```

Comment:
TLDR: This file essentially creates an AWS configuration that automatically generates an AWS EC2 SPOT Instance with specified configurations, alleviating concerns about EC2 Spot Interruptions while still benefiting from cost-effective pricing.